playground/package.json

```diff
@@ -14,12 +14,9 @@
       "localforage": "npm:localforage@^1.2.2",
       "lodash.throttle": "npm:lodash.throttle@^4.1.1",
       "object-assign": "npm:object-assign@^4.1.1",
-      "postcss": "npm:postcss@^6.0.8",
-      "postcss-css-variables": "npm:postcss-css-variables@0.14.0",
-      "postcss-inline-comment": "npm:postcss-inline-comment@^3.0.0",
```

Comment:
Removed `postcss-inline-comment` because it doesn't have a new version that supports `postcss@^8`